src/manifest.json

```diff
@@ -15,7 +15,7 @@
   },
   "content_scripts": [
     {
-      "matches": ["http://*/*", "https://*/*", "<all_urls>"],
+      "matches": ["http://*.tiktok.com/*", "https://*.tiktok.com/*"],
```

Comment:
i like manifest v3 but seemed like it's gonna be a hassle supporting it across browsers: https://blog.mozilla.org/addons/2021/05/27/manifest-v3-update/

> We are also interested in hearing about specific use cases we should keep in mind so that your extension will be compatible with Chrome for Manifest V3.

we should discuss this a little bit more